back-end/tests/endpoints.test.js

```diff
@@ -1,10 +1,11 @@
 /* eslint-disable camelcase */
-
-const app = require("../server.js");
+require("../environment");
+const app = require("../application")("production");
```

Comment:
Loading the `production` env in your tests is... reckless!

I would expect to see something like:
```js
const app = require("../application")("test");
```

Reply:
you are right, I was using that to make sure it worked initially. Good catch!

Reply:
But as I hinted elsewhere, I don't think the `application.js` module should return a function having the environment as an input parameter. I think the exported function in question could be environment-agnostic.